client/js/zip/z-worker.js

```diff
@@ -0,0 +1,153 @@
+/* jshint worker:true */
+(function main(global) {
+	"use strict";
+
+	if (global.zWorkerInitialized)
+		throw new Error('z-worker.js should be run only once');
+	global.zWorkerInitialized = true;
```

Comment:
This is 150+ lines with vague/no comments. please add comments to `z-worker.js` bc I'm illiterate without comments

Reply:
This file is part of zip.js. I didn't even write this; like I said, it's a library that I just copied into the repo because there is no CDN to serve it. You do not have to review any files in the `zip/` folder, because I did not write them and will not be editing them either.

Reply:
Okay I see, I was confused.